.cirrus.yml

```diff
@@ -13,7 +13,7 @@ task:
   name: Lint Api
   container:
     image: golangci/golangci-lint:latest
-  lint_script: golangci-lint run -v --out-format json > lint-report.json
+  lint_script: golangci-lint run --timeout=10m -v --out-format json > lint-report.json
```

Comment:
Good catch!

Reply:
actually, I don't know why it got so slow TBH